src/store/user-store.js

```diff
@@ -0,0 +1,190 @@
+import { create } from "zustand";
+
+const BASE_URL = "http://localhost:3000";
+
+const useUserStore = create((set, get) => ({
+  user: null, // state
+
+  loginSeeker: async (email, pass) => {
+    const endpoint = `${BASE_URL}/login/seeker`;
+
+    try {
+      const response = await fetch(endpoint, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ email, pass }),
+      });
+
+      if (!response.ok) {
+        throw new Error("Sign in request failed");
+      }
+
+      const data = await response.json();
+      if (!data.jwt) {
+        throw new Error("Failed to log in as seeker");
+      }
+
+      localStorage.setItem("jwt", data.jwt);
+
+      const userData = await get().fetchSeeker(data.jwt);
+      console.log("user data", userData);
```

Comment:
we can remove the console.logs for loginSeeker, fetchSeeker, or both, it's running twice

Reply:
adn possibly rest too